tools/gn

```diff
@@ -147,15 +147,21 @@ def to_gn_args(args):
     gn_args['skia_use_wuffs'] = True
```

Comment:
If possible, it would be good to group wasm-dependent logic so there's not too much repeated `if args.target_os != 'wasm'` in the code, but we should be careful about the order of operations as this file both writes and reads from `gn_args`.

Reply:
Done.